File: bin/check-api.pl
#!/usr/bin/perl

use warnings;
use strict;

use FindBin qw($Bin);
use lib "$Bin/ZabbixAPI";

use Data::Dumper;
use ZabbixAPI;
use Getopt::Long;
my $username = 'Admin';
my $password = 'zabbix';
my $api_url = 'http://localhost/api_jsonrpc.php';

my $zbx;
my $params;
my $json;
my $result;

GetOptions(
    "api_url=s" =>  \$api_url,
    "password|p=s"       => \$password,
    "username|u=s"      =>   \$username
) or die("Error in command line arguments\n");



$zbx = ZabbixAPI->new( { api_url=>$api_url, username => $username, password => $password } );
eval {
	$zbx->login();
};
if ($@) {
	die $@;
}
else {
	print "Zabbix API is ready\n";
	$zbx->logout();
}

File: bin/check-api.py
#!/usr/bin/env python
# -*- coding: UTF-8 -*-
import sys
from zabbix.api import ZabbixAPI
from zabbix_cli import zabbix_arg_parse

args = zabbix_arg_parse()
# Create ZabbixAPI class instance
try:
    zapi = ZabbixAPI(url=args.api_url,
                     user=args.username,
                     password=args.password)
except:
    sys.exit("Zabbix API is not available")
else:
    print("Zabbix API is ready")
    zapi.user.logout()
